src/app/pages/common/entity/table/table.component.ts

```diff
@@ -253,7 +253,7 @@ export class TableComponent implements OnInit, AfterViewInit, AfterViewChecked {
     } else {
       this.tableService.delete(this, row);
     }
-    event.stopPropagation();
+    event.stopPropagation(); // eslint-disable-line no-restricted-globals
```

Comment:
I took one of these lines and just removed event.stopPropagation() and couldn't see any difference.
I would not be surprised if this is not needed, otherwise perhaps the original intend was to pass event in the function.